build/publish-mrt.yml

```diff
@@ -128,6 +128,10 @@ steps:
     TargetFolder: '$(Build.ArtifactStagingDirectory)\fullnuget\build'
     flattenFolders: false
 
+- script: |
```

Comment:
A better display name for these we if we are keeping these in?

Reply:
Snicker. These should have been commented out. Thanks.

I'm planning a future PR removing all these (and other commented-out dead code) and add a separate README.md or something that includes these (and other) debugging tips.